frontend/ui/widget/menu.lua

```diff
@@ -457,8 +457,8 @@ end
 function MenuItem:getGesPosition(ges)
     local dimen = self[1].dimen
     return {
-        x = (ges.pos.x - dimen.x)/dimen.w,
-        y = (ges.pos.y - dimen.y)/dimen.h,
+        x = math.floor((ges.pos.x - dimen.x) / dimen.w),
+        y = math.floor((ges.pos.y - dimen.y) / dimen.h),
```

Comment:
math.floor() abuse ! (This one feels like déja vu :)
We expect a float between 0 and 1, so we can:
https://github.com/koreader/koreader/blob/03885071b97b63d9025871de2f4c16744056f0be/frontend/apps/reader/modules/readertoc.lua#L715-L730
Currently, in the TOC, you can't follow a TOC entry that is expand'collaps'able by tapping on its right half: it will expand or collapse when tapping anywhere on its width.

Quick check looking at math.floor with git log -p over the commits from February, and it seems this is the only one that is a ratio and should not be math.floor()ed.

Reply:
I knew this felt funky the moment I did it, thanks ^